src/templates/app/cashier/paymentagent_withdraw.jsx

```diff
@@ -72,27 +114,14 @@ const PaymentAgentWithdraw = () => (
                             <SeparatorLine className='gr-padding-10' show_mobile />
                         </div>
                         <FormRow label={it.L('Amount')} id='txtAmount' type='text' />
-                        <FormRow
-                            label={it.L('Further instructions')}
-                            id='txtDescription'
-                            type='textarea'
-                            attributes={{ rows: 4, cols: 19, maxLength: 250 }}
-                        />
                         <SubmitButton msg_id='withdrawFormMessage' type='submit' text={it.L('Submit')} />
+                        <div className='gr-8 gr-centered gr-padding-20'>
+                            {it.L('Notes: [_1] does not charge any transfer fees.', it.website_name)}
```

Comment:
isn't this a single note? why notes?

Reply:
I followed the design, but it doesn't make sense, so I changed it.